cg/meta/workflow/rnafusion.py

```diff
@@ -0,0 +1,315 @@
+                fusioncatcher=fusioncatcher,
+                arriba=arriba,
+            )
+        )
+        nextflow_options = self.__build_command_str(
+            self.get_verified_arguments_nextflow(
+                case_id=case_id,
+                log=log,
+            )
+        )
+        command = ["run", self.nfcore_pipeline_path]
+        parameters = (
+            nextflow_options
+            + ["-bg", "-q"]
+            + command
+            + options
+            + [
+                " > "
+                + str(self.get_case_path(case_id))
+                + "/"
+                + case_id
+                + "-stdout.log 2> "
+                + str(self.get_case_path(case_id))
+                + "/"
+                + case_id
+                + "-stdout.err  < /dev/null & "
+            ]
+        )
+        self.process.run_command(parameters=parameters, dry_run=dry_run)
+
+    def get_deliverables_file_path(self, case_id: str) -> Path:
+        """Returns a path where the rnafusion deliverables file for the case_id should be located.
+
+        (Optional) Checks if deliverables file exists
+        """
+        return Path(self.get_case_path(case_id), case_id + "_deliverables.yaml")
+
+    def get_template_deliverables_file(self, rnafusion_bundle_template: Path) -> pd.DataFrame:
+        """Read deliverables file template and output to dataframe format"""
+        return pd.read_csv(rnafusion_bundle_template)
+
+    def edit_template_deliverables_file(
+        self, case_id: str, deliverables_template: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Replace PATHTOCASE and CASEID from template deliverables file to corresponding strings, add path_index column"""
+        edited_deliverables = deliverables_template.replace(
+            {"PATHTOCASE": str(self.get_case_path(case_id))}, regex=True
+        )
+        edited_deliverables = edited_deliverables.replace({"CASEID": case_id}, regex=True)
+        edited_deliverables = edited_deliverables["path_index"] = "~"
+        return edited_deliverables
+
+    def convert_deliverables_dataframe_to_dict(self, dataframe: pd.DataFrame):
+        """Convert deliverables dataframe to dict with dict values in list format  [{column -> value}, … , {column -> value}] with (orient="records"). Also reformat apostrophes in path_index column"""
+        return dataframe.to_dict(orient="records").replace("'~'", "~")
+
+    def report_deliver(self, case_id: str) -> None:
+        """Get a deliverables file template from resources, edit by replacing paths and case_id, then write deliverables file"""
+        deliverables_template: pd.DataFrame = self.get_template_deliverables_file(
+            resources.rnafusion_bundle_filenames_path
+        )
+        edited_deliverables: pd.DataFrame = self.edit_template_deliverables_file(
+            case_id, deliverables_template
+        )
+        deliverables_file_dict: dict = self.convert_deliverables_dataframe_to_dict(
```

Comment:
```suggestion
        deliverables_file: dict = self.convert_deliverables_dataframe_to_dict(
```
You do not have to add "dict" in the variable name - it is already recorded in the type hint and thus in your editor